### DISCUSSION TITLE
ver.0.0.0 개선 사항

### DISCUSSION THREAD

**Discussion by hangooksaram** [dailry/dailry#312] [General]
다일리 를 ver.0 개선하면 좋을 것 같다는 생각이 들었습니다. 따라서 `개선 목표` 와 `개선 유형` 을 먼저 정의 하고, 그에 따른 `개선 사항 정리` 를 해보려고 합니다.

## 개선 목표
**유저가 ver.0 기능을 문제 없이 사용하는 것** 

### 버그 수정
보통 `기능` 관련한 항목 일 듯합니다. 원래 기능대로 동작하지 않는 부분을 수정합니다.

### 사용성 향상
`UI/UX` 관련한 항목 일 듯합니다. 유저가 사용할 때 어려운 점을 해결 하는 방향으로 생각하면 좋을 듯 합니다.

## 개선 사항 정리
- **댓글** 로 작성합니다. 
- `개선 유형` 에 따라 나누어 정리합니다.

---
### 버그 수정

- [x] : 액세스 만료시 리프레시 토큰 동작이 잘 안되는듯 함(수현)
    - dev에 머지는 되어있는데, main에 아직 반영이 안되어있는 상태. dev에서 한 번 테스트해보면 될 것 같음.(수현)
- [x] : **비 로그인 시 다일리 만들기 클릭 시 빈 화면 (하경)**
    - 로그인 페이지로 리다이렉트하면 좋을 것 같아요.
- [x] : 다이어리 공유 시 작성한 해시태그와 상관없이 모든 해시태그가 #일반으로 작성됨 (하경)
- [x] : 공개 다일리 게시글 삭제는 되는데 수정이 안되는 것 같음. (하경)

### 사용성 향상

- [x] : **메인 홈페이지 화면 비율 변경해도 가려지지 않도록 수정 (하경)**
- [x] : **로그인 하게 되면 메인홈페이지 로그인 -> 로그아웃으로 변경되도록 수정 (하경)**
- [x] : 공개 다일리 이름 변경 (커뮤니티 같이 이해하기 쉬운 이름으로 변경하면 좋을 듯) (하경)
- [x] : **첫 페이지 생성 시, 새로고침 해야 생성된 페이지 확인 가능 (하경)**
- [x] : 닉네임 변경시 중복닉네임일 시 알림 필요 (하경)
- [x] : 비밀번호 변경시 현재 비밀번호가 틀리면 알림 필요 (하경)
- [x] : 새 다일리 추가시에나 이름 수정시 내용물이 양옆으로 삐져나옴 (하경)
- [x] : 마이페이지 접속시 메인 화면이나 뒤로가기 할 수 있는 버튼 필요. (하경)
- [x] : 홈 페이지 에 마이페이지 버튼 추가 (현재)
- [x] : 다일리 페이지 공유하기 버튼 추가 (현재)
- [x] : 다일리 저장 버튼 아이콘 추가 (현재)
- [x] : 페이지 툴바 버튼 hover 시 해당 버튼이 어떤 기능을 하는지 알려주는 툴팁 (현재)

### 깡통기능 처리
- [x] : 메인 화면에 팀 다일리를 눌렀을 때 아무런 반응이 없는데 github 링크를 달면 좋을 거 같음 (하경)
- [x] : 커뮤니티 좋아요 기능 (건우, 하경)
- [ ] : 커뮤니티 최신순/인기순 정렬 (건우, 하경)
- [ ] : 공개 다일리 페이지 DailryHot (건우, 하경)
- [ ] : **스티커 기능 추가 (현재)**

**Comment by gomudayya** [comment]
사용성 향상

- 메인 홈페이지 화면 비율 변경해도 가려지지 않도록 수정
- 로그인 하게 되면 메인홈페이지 로그인 -> 로그아웃으로 변경되도록 수정

기능

- 액세스 만료시 리프레시 토큰 동작이 잘 안되는듯 함. 수정필요

**Comment by hangooksaram** [comment]
### 버그 수정
- 화면 비율에 따라 첫 화면의 메뉴 보이지 않음
- 비 로그인 시 `다일리 만들기` 클릭 시 빈 화면
- 다일리 `저장 버튼 아이콘` 이 `내보내기 아이콘` 과 동일 

### 사용성 향상
- 공개 다일리 이름 변경 (커뮤니티 같이 이해하기 쉬운 이름으로 변경하면 좋을 듯)   
- 첫 페이지 생성 시, 새로고침 해야 생성된 페이지 확인 가능 
- 페이지 툴바 버튼 hover 시 해당 버튼이 어떤 기능을 하는지 알려주는 툴팁

**Comment by SSUHYUNKIM** [comment]
### 사용성 향상
- 마이페이지가 확실하지 않아서 마이페이지 버튼이 따로 있으면 좋겠음
- 공유하기 버튼을 찾기가 어려워서 따로 아이콘이 있으면 좋겠음
- 메인 화면에 팀 다일리를 눌렀을 때 아무런 반응이 없는데 github 링크를 달면 좋을 거 같음

### 기능
- 사진 추가?(다일리 페이지 4번째 버튼) 누르고 페이지를 누르면 흰 화면이 뜨는데 수정 필요.
- 다이어리 공유 시 작성한 해시태그와 상관없이 모든 해시태그가 #일반으로 작성됨
- 공개 다일리 페이지에 DailryHot이 동작은 안하고 그냥 #수능으로 되어있는데 기능을 추가하거나 삭제하거나 해야 할 거 같음.
- 마이페이지 접속시 메인 화면이나 뒤로가기 할 수 있는 버튼 필요.
- 공개 다일리 게시글 삭제는 되는데 수정이 안되는 것 같음.

**Comment by puranium235** [comment]
### 기능
- 커뮤니티 좋아요 기능이 안됨
- 커뮤니티 최신순/인기순 정렬

### 사용성 향상
- 닉네임 변경시 중복닉네임이면 아무 반응이 없음 (콘솔에 뜸)
- 비밀번호 변경도 마찬가지
- 새 다일리 추가시에나 이름 수정시 내용물이 양옆으로 삐져나옴
